### ISSUE TITLE
Request for new API methods

### ISSUE THREAD

**Issue by sibanand-cis** [autopilotdev/autopilotdev.github.io#2]
1.) To get all Unsubscribes contact.
2. To get all removed contact from any particular List.
3.) while fetching all the contacts which belong from particular list, it return "created_date" that is a date on which contact has been created, but I want a date at which contact has been added to List.
=> Returning "contact created date"
=> I want "contact added to list date"

**Comment by chrissharkey** [comment]
Hi Sibanand,

I changed the title of your ticket to "Request for new API methods" as that is really what this is.

1) Getting all unsubscribed contacts - this is interesting and we will consider adding it.

2) We will not be adding the ability to get contacts who have been removed from a list. To accomplish this simply create a journey with a "Removed from list" trigger and connect it to an "Add to list" action. This will get you a list of contacts removed from that this.

3) Getting the date contacts are added to a list - this is interesting and we will consider adding it.

We will be looking at adding new methods to our API before the end of the year. I will update this ticket if we decide to add either of these methods.
